pkg/cloud/services/compute/instance.go

```diff
@@ -18,11 +18,14 @@ package compute
 
 import (
 	"fmt"
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/networks"
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/subnets"
+	"sigs.k8s.io/cluster-api-provider-openstack/pkg/cloud/services/networking"
```

Comment:
I'd suggest moving this imports down into the appropriate groups below

Reply:
Done